Onboarding/engineering-best-practices.md

```diff
@@ -39,10 +39,10 @@ Here is a brief list of Veteran Services Platform (VSP) recommended engineering
 ### Backend development
 
 - Discuss your implementation ideas with VSP backend team members early and often. There are many common patterns and classes that are in place in Vets-API that should be leveraged, that we can point you to.
-- RuboCop is used for Ruby style linting, and is enforced at the CI level.
-- [Screencast](https://youtu.be/R7F4BruOAwo) on patterns and practices when adding an endpoint to Vets-API (30min.)
+- [RuboCop](https://github.com/rubocop-hq/rubocop) is used for Ruby style linting, and is enforced at the CI level.
```

Comment:
Should we mention `RSpec` is used for testing here?  We strive for 100% coverage, and 90% coverage is enforced at the CI level.